targets/TARGET_Cypress/TARGET_PSOC6/cy_port_api.c

```diff
@@ -0,0 +1,75 @@
+/* mbed Microcontroller Library
+ * Copyright (c) 2006-2013 ARM Limited
```

Comment:
these new files should be 2019 (wrong copy-paste): ` * Copyright (c) 2019, Arm Limited and affiliates.` would be perfect

Reply:
https://github.com/ARMmbed/mbed-os/pull/10692/commits/d77fb26e9f2d2f67fd3c6802ead4915e4dc11642: adjusted the copyright of new cy_*_api.c files, per the proposed template.